.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,3 +12,13 @@
 ### Related Tickets
 
 - [#ticket_number](https://hipo.codebasehq.com/projects/PROJECT_SLUG/tickets/ticket_number)
+
+### Checklist:
+
+* ☐ Cleaned up the console log and browsed to see if any errors or warnings popped up
+* ☐ Checked for forgotten todo comments
```

Comment:
I don't think TODO comments needs to be added to the checklist

Reply:
Makes sense, should we remove the FIXME line also, wdyt?

Reply:
I use TODO for technical debt kinda messages but I also started using `FIXME` when I'm developing and there is something that I want to come back to and change before PR is merged. So I think `FIXME` can still be included in the checklist